readme.md

```diff
@@ -119,26 +126,32 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** There are a couple different ways to accomplish this goal. Which method did you choose?
 
 * **Your Answer:**
-
+async await method.
```

Comment:
The question here is about what Mongoose method to use. For example, you can use `.find()` or `.findById()`.